.github/workflows/build.yml

```diff
@@ -213,19 +215,32 @@ jobs:
       run: |
         mkdir build
         cd build
-        cmake .. ${{ env.BUILD_FLAGS }} -DCMAKE_BUILD_TYPE=${{ env.BUILD_MODE }} -DCMAKE_C_COMPILER=/usr/local/opt/llvm@14/bin/clang -DCMAKE_CXX_COMPILER=/usr/local/opt/llvm@14/bin/clang++ -G Ninja
+        cmake .. ${{ env.BUILD_FLAGS }} -DCMAKE_BUILD_TYPE=${{ env.BUILD_MODE }} -DPORTABLE=OFF -DMACOS_ENABLE_BUNDLE=ON -DCMAKE_C_COMPILER=/usr/local/opt/llvm@14/bin/clang -DCMAKE_CXX_COMPILER=/usr/local/opt/llvm@14/bin/clang++ -G Ninja
         
     - name: "Build Cemu"
       run: |
         cmake --build build
 
     - name: Prepare artifact
       if: ${{ inputs.deploymode == 'release' }}
-      run: chmod a+x bin/Cemu_release && mv bin/Cemu_release bin/Cemu
+      run: |
+        mkdir bin/Cemu
+        mv bin/Cemu_release.app bin/Cemu/Cemu.app
+        mv bin/Cemu/Cemu.app/Contents/MacOS/Cemu_release bin/Cemu/Cemu.app/Contents/MacOS/Cemu
+        sed -i '' 's/Cemu_release/Cemu/g' bin/Cemu/Cemu.app/Contents/Info.plist
+        mkdir bin/Cemu/Cemu.app/Contents/SharedSupport
+        for folder in gameProfiles resources
+        do
+          cp -R ./bin/${folder} ./bin/Cemu/Cemu.app/Contents/SharedSupport
```

Comment:
Looks like this part is still dependent on GitHub Actions. It would be best if this (and the stuff with MoltenVK below) are done with CMake so that builds can be produced locally.

Reply:
Done :) not too familiar with cmake but it seems to be working